assessment_store/scripts/derive_assessment_values.py

```diff
@@ -0,0 +1,53 @@
+import click
+
+from application_store.db.queries.application.queries import get_application
+from assessment_store.db.models.assessment_record.assessment_records import AssessmentRecord
+from assessment_store.db.queries.assessment_records._helpers import derive_application_values
+
+
+@click.command()
+@click.option(
+    "-a",
+    "--application_id",
+    help="Application ID to derive values",
+    prompt=True,
+)
+# @click.option(
+#     "-c",
+#     "--commit",
+#     is_flag=True,
+#     default=False,
+#     show_default=True,
+#     help="Whether to commit the derived values to assessment_store",
+#     prompt=False,
+# )
+def derive_assessment_values(application_id):
+    from flask import current_app
+
+    from db import db
+
+    current_app.logger.info(
+        "Deriving values for {application_id}",
+        extra=dict(application_id=application_id),
+    )
+
+    application_json = get_application(application_id, include_forms=True, as_json=True)
+    derived_values = derive_application_values(application_json)
+    current_app.logger.info("Derived the following values {derived_values}", extra=dict(derived_values=derived_values))
```

Comment:
could we pretty-print this rather than console log it maybe? Might make it easier to review the derived values if they're multi-line/indented, rather than a single long line?

Reply:
Agree, done